atlante/grids/postgresql/postgresql.go

```diff
@@ -176,6 +177,12 @@ func NewGridProvider(config grids.ProviderConfig) (grids.Provider, error) {
 	queryLngLat, _ = config.String(ConfigKeyQueryLngLat, &queryLngLat)
 	var queryMDGID string
 	queryMDGID, _ = config.String(ConfigKeyQueryMDGID, &queryMDGID)
+	if (queryLngLat != "" || queryMDGID != "") && (queryLngLat == "" || queryMDGID == "") {
+		if queryLngLat == "" {
+			return nil, errors.String("error " + ConfigKeyQueryLngLat + " not set, when " + ConfigKeyQueryMDGID + " is set")
```

Comment:
maybe `errors` needs something similar to fmt.Errorf(...)?

Reply:
There is a reason I did it this way. This will generate a constant string, instead of a string that has to be interpolated.